packages/mdl-snackbar/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "mdl-snackbar",
```

Comment:
This needs a simple `"description"` (e.g. "A material design snackbar component"), as well as a `
main"` key that points to `"./index.js"`.